cumulusci/tasks/github/publish.py

```diff
@@ -54,13 +53,8 @@ def _init_options(self, kwargs):
 
         self.options["renames"] = self._process_renames(self.options.get("renames", []))
 
-        if self.options.get("version") in ("latest", "latest_beta"):
-            get_beta = self.options["version"] == "latest_beta"
-            self.options["version"] = str(
-                self.project_config.get_latest_version(beta=get_beta)
-            )
-        if "ref" not in self.options and "version" not in self.options:
-            raise TaskOptionsError("Either `ref` or `version` option is required")
```

Comment:
@Br4nd0R I guess we have to stop supporting `version` set to a specific version number, but the [main use of this in the wild](https://github.com/SalesforceFoundation/sal/blob/a12a3360fe9f7d9b6f07ce52255b003384d050d1/cumulusci.yml#L1159) is using `version: latest` and I don't think we need or want to stop supporting `version: latest` and `version: latest_beta`. Instead of the existing logic (get version using get_latest_version, then convert to a tag name using get_tag_for_version) we can simply use get_latest_tag which also takes a flag for whether to find beta or final tags. /cc @jstvz